Sources/Utilities/KMTextToSpeech.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  KMTextToSpeechHandler.swift
+//  KommunicateChatUI-iOS-SDK
+//
+//  Created by sathyan elangovan on 29/07/22.
+//
+
+import Foundation
+import AVFoundation
+import KommunicateCore_iOS_SDK
+
+// To handle Text To Speech in the conversation
+class KMTextToSpeech : NSObject, AVSpeechSynthesizerDelegate  {
+    let synthesizer : AVSpeechSynthesizer = AVSpeechSynthesizer()
+    var index = 0
+    var speechStarted = false
+   
+    public override init() {
+        super.init()
+        synthesizer.delegate = self
+    }
+    
+    public static let shared = KMTextToSpeech()
+    var messageModels : [ALMessage] = []
```

Comment:
Change the name of this var to `messageQueue`. Also, change index to `messagesProcessed` or something.

Reply:
done